src/extension.js

```diff
@@ -41,7 +41,7 @@ const ServiceToggle = GObject.registerClass({
 
     _init() {
         super._init({
-            label: 'GSConnect',
+            title: 'GSConnect',
```

Comment:
This breaks compatibility for GNOME 43

Reply:
I didn't test yet, thanks for the testing help! In general we only support the latest shell version, so  the correct solution is to drop 43 from metadata.json

Reply:
Thanks for catching that, I totally missed it. @daniellandau If you have the time to fix the missing icon feel free - otherwise I'll try to come back to this when I have time. :)